packages/js/e2e-utils-playwright/src/editor.js

```diff
@@ -0,0 +1,164 @@
+const { expect } = require( '@playwright/test' );
+
+export const closeChoosePatternModal = async ( { page } ) => {
+	const closeModal = page
+		.getByLabel( 'Scrollable section' )
+		.filter()
+		.getByRole( 'button', {
+			name: 'Close',
+			exact: true,
+		} );
+	await page.addLocatorHandler( closeModal, async () => {
+		await closeModal.click();
+	} );
+};
+
+export const disableWelcomeModal = async ( { page } ) => {
+	// Further info: https://github.com/woocommerce/woocommerce/pull/45856/
+	await page.waitForLoadState( 'domcontentloaded' );
+
+	const isWelcomeGuideActive = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' )
+	);
+
+	if ( isWelcomeGuideActive ) {
+		await page.evaluate( () =>
+			wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' )
+		);
+	}
+};
+
+export const openEditorSettings = async ( { page } ) => {
+	// Open Settings sidebar if closed
+	if ( await page.getByLabel( 'Editor Settings' ).isVisible() ) {
+		console.log( 'Editor Settings is open, skipping action.' );
+	} else {
+		await page.getByLabel( 'Settings', { exact: true } ).click();
+	}
+};
+
+export const getCanvas = async ( page ) => {
+	return page.frame( 'editor-canvas' ) || page;
+};
+
+export const goToPageEditor = async ( { page } ) => {
+	const responsePromise = page.waitForResponse(
+		( response ) =>
+			response.url().includes( '//page' ) && response.status() === 200
+	);
+	await page.goto( 'wp-admin/post-new.php?post_type=page' );
+	await disableWelcomeModal( { page } );
+	await closeChoosePatternModal( { page } );
+	await responsePromise;
+};
+
+export const goToPostEditor = async ( { page } ) => {
+	const responsePromise = page.waitForResponse(
+		( response ) =>
+			response.url().includes( '//single' ) && response.status() === 200
+	);
+	await page.goto( 'wp-admin/post-new.php' );
+	await disableWelcomeModal( { page } );
+	await responsePromise;
+};
+
+export const fillPageTitle = async ( page, title ) => {
```

Comment:
I'm not crazy about this utility.  It's not a real savings and just obfuscates what's already simple built-in methods.